changelogs/unreleased/gh-6986-introduce-sql-expr-functions.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
```

Comment:
Seems like rather an SQL feature, not box.

Reply:
Fixed.